scripts/generate-babel-types-docs.js

```diff
@@ -37,6 +37,10 @@ function getType(validator) {
     return validator.type;
   } else if (validator.oneOfNodeTypes) {
     return validator.oneOfNodeTypes.join(' | ');
+  } else if (validator.oneOfNodeOrValueTypes) {
+    return validator.oneOfNodeOrValueTypes.join(' | ');
+  } else if (validator.oneOf) {
+    return validator.oneOf.map(val => util.inspect(val)).join(' | ');
```

Comment:
Does this path never get run? It's passing all tests (like node 0.10) and this file isn't transpiled I don't think and there's an arrow function